### ISSUE TITLE
SyncTeX support

### ISSUE THREAD

**Issue by rekka** [tectonic-typesetting/tectonic#49]
Currently there does not seem to be a way to enable SyncTeX, which is rather convenient for interactive editing.

**Comment by pkgw** [comment]
Yes. As you may have seen, the underlying code is there, but I have never tried to run it.

One could imagine adding an option to the command-line client that enables SyncTeX in the engine. But I don't want to end up with an enormous proliferation of command-line options to twiddle various knobs in the engine. Something to think about.

**Comment by rekka** [comment]
Thanks for the answer. I guess adding `--synctex` command-line option that other engines have would be reasonable.

I noticed the following at the top of `synctex.c` file, wasn't sure if it means that there might be some deeper issues:

```
 * Note: these settings used to depend on the no_pdf_output variable. Tectonic
 * fixes this to 1, but we do then generally run xdvipdfmx to produce a PDF.
 * Need to investigate how to deal with this.
```

Interestingly, adding `\synctex=1` to the tex file preamble does make tectonic produce a synctex file, but it does not seem to work in a pdf viewer.

**Comment by rekka** [comment]
I have fixed at least `\synctex=1` in #56 , which now produces a working file.

**Comment by pkgw** [comment]
Nice! Thank you for working on this.
